apps/heureka/src/components/shared/ListController.jsx

```diff
@@ -5,28 +5,18 @@
 
 import React, { useEffect, useMemo, useState } from "react"
 import { useQuery } from "@tanstack/react-query"
-import {
-  useGlobalsQueryClientFnReady,
-  useGlobalsQueryOptions,
-  useGlobalsActions,
-  useActiveFilters,
-  usePredefinedFilters,
-  useGlobalsActiveNavEntry,
-  useSearchTerm,
-} from "../../hooks/useAppStore"
+import { useGlobalsQueryClientFnReady, useGlobalsQueryOptions, useGlobalsActions } from "../../hooks/useAppStore"
 import { Pagination, Container, Stack } from "@cloudoperators/juno-ui-components"
 import { useActions as messageActions } from "@cloudoperators/juno-messages-provider"
 import { parseError } from "../../helpers"
+import { useGlobalsActiveNavEntry } from "../../hooks/useAppStore"
 
-const ListController = ({ queryKey, entityName, ListComponent }) => {
+const ListController = ({ queryKey, entityName, ListComponent, activeFilters, searchTerm, enableSearchAndFilter }) => {
```

Comment:
`entityName`still being used?

Reply:
I can pass the necessary functions that use entityName here, allowing its removal from this controller.